src/utils.ts

```diff
@@ -355,6 +387,13 @@ export function parseRadicleId(urn: string): string {
   return urn.replace(/^rad:[a-z]+:/, "");
 }
 
+// Get amount of days passed between two dates without including the end date
+export function getDaysPassed(from: Date, to: Date): number {
+  return Math.floor(
+    (to.getTime() - from.getTime()) / (24 * 60 * 60 * 1000)
+  );
+}
```

Comment:
Is this used?

Reply:
Yes, we use this function in the commit grouping by week here
https://github.com/radicle-dev/radicle-interface/blob/c2e52ddbea05496c0f3a13879d73068e2793d4f5/src/commit.ts#L167

Since we are using it only once, we could also move it into `commit.ts`